fe/pom.xml

```diff
@@ -717,6 +717,13 @@ under the License.
                 <version>42.4.4</version>
             </dependency>
 
+            <!-- https://mvnrepository.com/artifact/com.oracle.database.jdbc/ojdbc8 -->
+            <dependency>
+                <groupId>com.oracle.database.jdbc</groupId>
+                <artifactId>ojdbc8</artifactId>
+                <version>12.2.0.1</version>
```

Comment:
Why not use a new version?

Reply:
It's fine to use the new version. The reason why we use this version here is that we use it every day. I will replace it.